public void testClose() throws IOException {
    shielded.close();
    assertFalse("closed", closed);
    try {
        shielded.write('x');
        fail("write(b)");
    } catch (IOException e) {
    }
    original.write('y');
    assertEquals(1, original.size());
    assertEquals('y', original.toByteArray()[0]);
}